### ISSUE TITLE
how to edit on my own images

### ISSUE THREAD

**Issue by flyingshan** [MrTornado24/IDE-3D#18]
I notice the advices from the section "Real portrait image editing", where first the following code should be executed:
```
python apps/infer_hybrid_encoder.py 
    --target_img /path/to/img_0.png
    --g_ckpt pretrained_models/ide3d-ffhq-64-512.pkl 
    --e_ckpt pretrained_models/encoder-base-hybrid.pkl
    --outdir out
```
But I found for my image, it does not have labels like those from `dataset.json`, so error occured:

>   File "/home/dianxin/hys/IDE-3D/apps/infer_hybrid_encoder.py", line 147, in <module>
>     c = [label_list[opts.target_img[-21:]]]
> KeyError: 'test.png'
> 

How can I generate the labels for my own images? Thank you!

**Comment by Bai426** [comment]
Yes, I found the same problem. The dataset.json is an empty file.
